test/parallel/test-crypto-hash.js

```diff
@@ -82,15 +82,35 @@ assert.deepStrictEqual(
 );
 
 // stream interface should produce the same result.
-assert.deepStrictEqual(a5, a3, 'stream interface is consistent');
-assert.deepStrictEqual(a6, a3, 'stream interface is consistent');
-assert.notStrictEqual(a7, undefined, 'no data should return data');
-assert.notStrictEqual(a8, undefined, 'empty string should generate data');
+assert.deepStrictEqual(
+  a5,
+  a3,
+  `Stream generated: ${a5.toString('hex')}, expected: ${a3.toString('hex')}`
+);
+assert.deepStrictEqual(
+  a6,
+  a3,
+  `Stream generated: ${a6.toString('hex')}, expected: ${a3.toString('hex')}`
+);
+assert.notStrictEqual(
+  a7,
+  undefined,
+  `Stream for no data generated: ${a7}, expected to generate value`
```

Comment:
a7 is always undefined in this case. So it is the same as having a static message. The same applies for the test below. I think it is best to just remove the message all along.